src/components/src/kepler-gl.tsx

```diff
@@ -420,17 +420,6 @@ function KeplerGlFactory(
     datasetsSelector = props => props.visState.datasets;
     filteredDatasetsSelector = createSelector(this.datasetsSelector, getVisibleDatasets);
 
-    availableProviders = createSelector(
-      (props: KeplerGLProps) => props.cloudProviders,
-      providers =>
-        Array.isArray(providers) && providers.length
-          ? {
-              hasStorage: providers.some(p => p.hasPrivateStorage()),
-              hasShare: providers.some(p => p.hasSharingUrl())
-            }
-          : {}
-    );
```

Comment:
we no longer need to inject providers since components can use the custom hook to access cloud list provider